### PR TITLE
Release v0.1

### PR DESCRIPTION
**Pull request by ilanolkies**
Releasing Data Vault `v0.1` 🥳 

I broke git history 😕 please check the release is OK...

### CODE REVIEW

File: modules/ipfs-cpinner-service/README.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+  </a>
+</p>
+
+A Centralized Data Vault service compatible with RIF Data Vault standard interface. It stores content in an IPFS node associated to a given DID and key.
+
+Alpha version at: []()
+
+## Features
+
+- API for storing, updating and deleting, accessible only proving DID control - uses DID Auth
```
Comment (miltonwg) [inline modules/ipfs-cpinner-service/README.md:20]:
```suggestion
- API for storing, updating and deleting, accessible only proving DID control - uses DID Auth
```

Add link to `DID Auth`?

File: modules/ipfs-cpinner-service/Dockerfile.dataVault [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM node:10
+
+WORKDIR /app
+
+COPY package.json ./
```
Comment (ilanolkies) [inline modules/ipfs-cpinner-service/Dockerfile.dataVault:5]: This is to have tsc inside the service? Shouldn’t we add tsc as dev inside the module?

File: modules/ipfs-cpinner-service/Dockerfile.dataVault [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+COPY package-lock.json ./
+
+RUN npm i
+
+COPY ./src ./src
```
Comment (ilanolkies) [inline modules/ipfs-cpinner-service/Dockerfile.dataVault:10]: And then have this file inside the service too

File: modules/ipfs-cpinner-service/Dockerfile.dataVault [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+COPY ./tsconfig.prod.json ./
+
+EXPOSE 5107
+
+CMD [ "npm", "run", "start:prod"]
```
Comment (ilanolkies) [inline modules/ipfs-cpinner-service/Dockerfile.dataVault:15]: Because this script will fail without the copy paste ceremony

**Comment by ilanolkies** [comment]
We can close this PR and rise an issue to improve the Docker flow. What do you think about the comments about Docker @lucasvuotto ?

**Comment by ilanolkies** [comment]
Closes #8

**Review by miltonwg — APPROVED** [review]

**Pull request merged**